pom.xml

```diff
@@ -528,6 +529,32 @@
                 </executions>
                 <inherited>false</inherited>
             </plugin>
+            <plugin>
+                <groupId>com.hazelcast.maven</groupId>
+                <artifactId>attribution-maven-plugin</artifactId>
```

Comment:
Where does this plugin come from?

Reply:
This is a new home-grown plugin that greps the dependency sources and searches for the "Copyright ..." lines there.
https://github.com/hazelcast/attribution-maven-plugin